openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx

```diff
@@ -278,12 +278,15 @@ export const getELKLayoutedElements = async (
       return {
         ...node,
         position: { x: layoutedNode?.x ?? 0, y: layoutedNode?.y ?? 0 },
+        // layoutedNode contains the total height of the node including the children height
+        // Needed to calculate the bounds height of the nodes in the export
+        height: layoutedNode?.height ?? node.height,
```

Comment:
I guess the nodes object already has height which is calculated based on the columns height. Can you confirm if we really need this new height. 
Also verify the spacing between nodes when column layer is active and in edit mode

Reply:
No, so we are calculating it but not passing or returning it to the nodes. Also verified the spacing in both column active and edit mode and they are fine.